rootfs/usr/local/bin/kopsctl

```diff
@@ -22,6 +22,17 @@ tasks:
     description: "Operations to perform on the kops cluster"
 
     tasks:
+      # kubecfg operationss
+      kubecfg:
+        description: "Kubecfg Management"
+        tasks:
+          # Writes out kubecfg
+          generate:
+            description: "Generate kubecfg"
+            script:
+            - *exit_on_errors
+            - generate-kubecfg
```

Comment:
in terms of naming, we have a lot of verbs... would prefer to consoidate on either `export` (what kops calls it) or `build` which is what we have for some of the other scripts (e.g. `build-kops-manifest`). That said, I do like `generate`, but alas we haven't used it yet so want to avoid introducing another verb.